website/src/views/covid.ts

```diff
@@ -25,9 +23,13 @@ const earliestDate = '2020-01-06';
 
 class CovidConstants {
     public readonly organism = Organisms.covid;
-    public readonly defaultDateRange: DateRange = 'last6Months';
+    public readonly defaultDateRange: DateRangeOption = {
```

Comment:
We could also use `DateRangeOption` from `@genspectrum/dashboard-components` (because ultimately, that's the type that counts)

Reply:
I now used it, and removed in the same step also the 'todayString'